### ISSUE TITLE
Linear Probes Training Issues

### ISSUE THREAD

**Issue by wimmerth** [PaulCouairon/JAFAR#5]
Could you please provide the scripts that can be used to replicate your linear probing experiments as described in Sec. 4?
More specifically, please provide the script for the multi-stage training for semantic segmentation, as described in Sec. 4.3.1.
Further, training the linear probe for depth estimation does not work as the necessary losses are undefined (see [train_probes.py](https://github.com/PaulCouairon/JAFAR/blob/f235bcea86cfe71e3326282bf5c0b75ae21579cd/evaluation/train_probes.py#L94)), probably removed during the code base cleaning. Please provide these losses so that the experimental results can be replicated.
Best, Thomas

**Comment by LoickCh** [comment]
Hi,
You also have to overwrite the optimizer learning rate to 5e-3, the number of epochs to 5, and to overwrite the batch size to 2 for JAFAR. It should be written in the paper, I will check later. The complete command we used is the following:

```
python evaluation/train_probes.py eval.task=depth optimizer.lr=5e-3 dataset_evaluation=coco backbone.name=vit_small_patch14_dinov2.lvd142m hydra.run.dir=evaluation/unsupervised/coco-depth/vit_small_patch14_dinov2.lvd142m-jafar model=jafar train_dataloader.batch_size=2 num_epochs=5
```

I have relaunched the command for bilinear and JAFAR and the results are coherent.

**Comment by LoickCh** [comment]
Hi,

We corrected a bug with the update of the learning rate in the scheduler during the eval. Should work fine now. Results can slightly differ from the paper, but should be close.
